staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -291,7 +291,12 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 		// Make sure our peer-packages are added and fully parsed.
 		for _, pp := range peerPkgs {
 			context.AddDir(pp)
-			getManualConversionFunctions(context, context.Universe[pp], manualConversions)
+			p := context.Universe[pp]
+			if nil == p {
+				glog.Errorf("failed to find pkg: %s ", pp)
```

Comment:
error or fatal? Seeing a number of fatals above.

Reply:
will change to fatal